service/src/main/java/com/expedia/alertmanager/service/conf/AppConfig.java

```diff
@@ -62,4 +63,11 @@ JestClientFactory clientFactory(ElasticSearchConfig elasticSearchConfig) {
     public KafkaTemplate<String, Alert> kafkaTemplate(KafkaConfig kafkaConfig) {
         return new KafkaTemplate<>(producerFactory(kafkaConfig));
     }
+
+    @Value("${mail.additional-validator-expression:}")
```

Comment:
Is the trailing colon correct?

Reply:
yes, 'mail.additional-validator-expression' defaults to empty string if not specified in the config

Reply:
OK